python/dlisio/ext/core.cpp

```diff
@@ -321,13 +321,15 @@ framesize frame_size(const char* fmt) {
         throw std::invalid_argument(fmt_msg);
     }
 
-    return framesize { src, dst };
+    return fdatasize { src, dst };
 }
 
-void read_all_fdata(const char* fmt,
-                    dl::stream& file,
-                    const std::vector< int >& indices,
-                    py::buffer dstb)
+void read_fdata(const char* pre_fmt,
```

Comment:
This is sweet, maybe it deserves its own commit ? (read_all_fdata -> read_fdata)

Reply:
To me that's the part of adding channel functionality. Change makes little sense without it, so I am not convinced. But we can discuss.